### ISSUE TITLE
A need for informational modifier for machines that indicates whether the crafting recipe will perform after player closes the UI or not.

### ISSUE THREAD

**Issue by Nextej** [loiste-interactive/Obenseuer-Issues#945]
Inclusion or exclusion of "Crude" modifier is not enough, since for instance a Geared Head Drill, despite not having a crude modifier, does require player's presence. So I propose "Manual" with description of "Requires a manual operator for the duration of crafting process" and "Self-Sufficient" with a description of "Once crafting is started, a manual operator is not required for the duration of that process" to be applied for all single crafting/machining/processing furniture

**Comment by Kastuk** [comment]
Yes, it's needful, even if sometimes its common sense, like for manual grinder.
